legal_db/static/styles.scss

```diff
@@ -1,5 +1,6 @@
 $color-tomato: rgb(237, 89, 47);
 $color-dark-turquoise: rgb(5, 181, 218);
```

Comment:
please remove this since it's no longer used

Reply:
ok i wil remove it